orca-clouddriver/src/test/groovy/com/netflix/spinnaker/orca/clouddriver/tasks/servergroup/CreateServerGroupTaskSpec.groovy

```diff
@@ -112,18 +112,18 @@ class CreateServerGroupTaskSpec extends Specification {
     then:
     _ * mortService.getAccountDetails("abc") >> [:]
     1 * katoService.requestOperations(operationCloudProvider, {
-      return it[createOpIndex]?.createServerGroup?.get(imageAttributeKey) == expectedImageId
+      return it[0]?.createServerGroup?.get(imageAttributeKey) == expectedImageId
```

Comment:
Can you please briefly explain why the index parameter can be removed?  I'm not following.